web/template/certificateprovider/you_have_decided_not_to_be_a_certificate_provider.gohtml

```diff
@@ -1,17 +1,18 @@
 {{ template "page" . }}
 
-{{ define "pageTitle" }}{{ tr .App "thankYou" }}{{ end }}
+{{ define "pageTitle" }}{{ trFormat .App "youHaveConfirmedYouDoNotWantToBeCertificateProvider" "DonorFullNamePossessive" (.App.Query.Get "donorFullNamePossessive") }}{{ end }}
```

Comment:
this should not include their name, just be generic (I know some personalised ones have slipped in, but we need to remove them at some point)

Reply:
I'd forgotten we'd agreed that...I'll revert to thank you 👍